<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NetworkChannels</class>
 <widget class="QDialog" name="NetworkChannels">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>280</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>280</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="L_NWC__NetworkChannelDialog">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Network Channels</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="T_NWC_NetworkChannels">
     <column>
      <property name="text">
       <string>Network</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Region</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Delete</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="NWC_DialogBox">
     <item>
      <layout class="QVBoxLayout" name="NWX_NetworkBox">
       <item>
        <widget class="QLabel" name="L_NWC_Network">
         <property name="text">
          <string>Network</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QComboBox" name="CB_NWC_Network"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="NWC_ChannelNameBOX">
       <item>
        <widget class="QLabel" name="L_NWC_ChannelName">
         <property name="text">
          <string>Channel Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="LE_NWC_ChannelName">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="NWC_ChannelIDBOX">
       <item>
        <widget class="QLabel" name="L_NWC_ChannelID">
         <property name="text">
          <string>Channel ID</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="LE_NWC_ChannelID">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="L_NWC_regions">
         <property name="text">
          <string>Region</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="CB_NWC_regions"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="PB_NWC_Add">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="NWC_ButtonBOX">
     <item>
      <widget class="QPushButton" name="PB_NWC_Save">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="PB_NWC_Cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
